webapp/src/js/controllers/contacts-content.js

```diff
@@ -106,10 +124,11 @@ angular.module('inboxControllers').controller('ContactsContentCtrl',
         $scope.setLoadingContent(id);
       }
 
-      return ContactViewModelGenerator(id, { getChildPlaces: !usersHomePlaceId || usersHomePlaceId !== id })
+      var options = { getChildPlaces: !usersHomePlaceId || usersHomePlaceId !== id };
+      return ContactViewModelGenerator.getContact(id, options)
```

Comment:
Moving `options` here was a bit of a hacky fix in order to make the `ContactViewModelGenerator` api more redux (/immutable) friendly, open to other suggestions.